docs/optimizing-with-codeflash/review-optimizations.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: "Staging Review for Optimizations"
+description: "Review and manage optimizations before creating pull requests with staging mode"
+icon: "layer-group"
+sidebarTitle: "Review Optimizations"
+keywords:
+  [
+    "staging review",
+    "optimization preview",
+    "pro feature",
+    "batch review",
+    "pull request management",
+  ]
+---
+
+# Staging Review for Optimizations
+
+<Note>
+  This is a Pro feature available exclusively to Codeflash Pro users. [Upgrade to
+  Pro](https://app.codeflash.ai/billing) to access it.
+</Note>
+Staging Review allows you to preview and evaluate all optimizations before creating pull requests. This
+feature provides a centralized review interface where you can examine proposed changes and selectively
+create pull requests for approved optimizations.
+
+## Benefits of Staging Review
+
+- **Preview without PRs:** Review all optimization suggestions without cluttering your repository with multiple pull requests
+- **Batch review:** Examine all optimizations in one centralized location
+- **Selective PR creation:** Choose which optimizations to convert into pull requests
+- **Reduced noise:** Keep your repository's PR list clean while evaluating changes
+
+## Using Staging Review
+
+To optimize your codebase with staging review enabled, run:
+
+```bash
+codeflash --all --staging-review
```

Comment:
We should add the function or file cmd reference as well with --staging-review arg as it can then be used by users for testing staging.